vagrant/bootstrap.sh

```diff
@@ -116,6 +116,7 @@ mkdir -p /var/{lib,log,run}/celery
 chown $RUN_AS_USER /var/{lib,log,run}/celery
 cp "${VAGRANT_PATH}"/celery.service /etc/systemd/system/
 cp "${VAGRANT_PATH}"/celery.conf /etc/
+cp "${VAGRANT_PATH}"/tmpfiles.d/celery.conf /usr/lib/tmpfiles.d/
```

Comment:
/ var/run/celery disappears when ubuntu starts up. And some functions of timesketch will not work. So, remake it.

Reply:
ack, good catch.